src/network/networkprotocol.h

```diff
@@ -224,9 +224,12 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 		Add TOCLIENT_MOVE_PLAYER_REL
 		Move default minimap from client-side C++ to server-side builtin Lua
 		[scheduled bump for 5.9.0]
+	PROTOCOL VERSION 45:
+		Minimap HUD element supports negative size values as percentages
+		[bump for 5.9.1]
```

Comment:
patch releases cannot bump the protocol version

Reply:
why not? a patch can very well require a network protocol extension (as is the case here). detecting a patch release and selectively applying workarounds is also something mods will be interested in.

anyways, going forward under the assumption that we don't bump for 5.9.1, it'd only really be viable to delay this to 5.10.0. which shouldn't be too bad given that we plan to release 5.10.0 in a few months.

Reply:
there is a scheduled bump for every minor release
so if we bump for 5.9.1 we need to somehow separately bump 5.10.0 by *two* versions.
not impossible, just a mess and we've never done this.

> it'd only really be viable to delay this to 5.10.0

that works.
other options are stupid workaround like "if it's the first minimap element and the size is 256x256 make it 25% instead"

Reply:
what problem would bumping the protocol version in 5.9.1 cause? early 5.10.0-dev and 5.9.1 would share the same protocol version, but that doesn't seem problematic.

Reply:
I can't think of a concrete problem but this inconsistency doesn't feel good to me.